pylabnet/utils/zi_hdawg_pulseblock_handler/zi_hdawg_pb_handler.py

```diff
@@ -0,0 +1,252 @@
+
```

Comment:
i think this belongs in hardware/ZI_HDAWG since it is totally hardware specific, but it's not a big issue if you prefer it here

Reply:
I had it in hardware/ZI_HDAWG first, but then though that after the separation of drivers and network functions, we would only want to keep the drivers in the hardware folder.